app/components/tabs_component.html.erb

```diff
@@ -1,6 +1,15 @@
-<div class="page-tab-container">
-  <% tabs.each do |tab| %>
-    <%= tab %>
-  <% end %>
-</div>
-<hr class="govuk-!-margin-top-0"/>
+<!--<div class="page-tab-container">-->
+  <%# tabs.each do |tab| %>
+    <%#= tab %>
+  <%# end %>
+<!--</div>-->
+<!--<hr class="govuk-!-margin-top-0"/>-->
```

Comment:
Hehe, are we sure we want to commit that?

Reply:
Also, `page-tab-container` can probably be cut from css now.

Reply:
hahah - yeah good spot

Reply:
Done